src/main/java/com/github/forax/umldoc/core/Call.java

```diff
@@ -51,6 +52,24 @@ record Group(Kind kind, List<Call> calls) implements Call {
      */
     public static final Group EMPTY_GROUP = new Group(Kind.NONE, List.of());
 
+    /**
+     * A method which returns the list of relevant calls for the sequence diagram.
+     * A call is relevant if its target is one of our entity.
+     *
+     * @param entitiesNames a {@link Set} containing every entity name
+     * @return the list of relevant Call
+     */
+    public List<Call> getRelevantCallsFromSet(Set<String> entitiesNames) {
```

Comment:
This implementation is quite ugly, but it's ok as a first rough cut

Reply:
#tryNotToCry
do you have any better idea ? Marius suggested to use the package name to know if the method is relevant, but we didn't manage to be sure if it could be a better idea